vts/plugins/scheme-cca-ssd-platform/main.go

```diff
@@ -355,7 +356,7 @@ func matchSoftware(evidence psatoken.IClaims, endorsements []Endorsements) bool
 			break
 		}
 
-		log.Printf("MeasType Evidence: %s, Endorsement: %s", *evComp.MeasurementType, attr.MeasType)
+		log.Debug("MeasType Evidence: %s, Endorsement: %s", *evComp.MeasurementType, attr.MeasType)
```

Comment:
pure quibbling: but having both `MeasType` and `MeasurementType` is quite puzzling :-)